src/_scss/pages/award/shared/awardAmountsSection/_charts.scss

```diff
@@ -419,8 +419,21 @@ $new-potential: #AAC6E2;
                     }
                 }
             }
-            .award-amounts-viz__desc.asst-nff-zero {
-                border-left: rem(5) solid $grants-line-color;
+            .award-amounts-viz__desc.asst-nff-zero,
+            .award-amounts-viz__desc.loan-file-c-outlay--zero,
+            .award-amounts-viz__desc.idv-file-c-outlay--z,
```

Comment:
Should this be `.award-amounts-viz__desc.idv-file-c-outlay--zero` rather than `.award-amounts-viz__desc.idv-file-c-outlay--z`?

Reply:
Shoot. Yes!